test/LTE_Tests/AD936x_LTETests.m

```diff
@@ -76,6 +76,16 @@ function ConfigHW(testCase, TxClass, RxClass)
     end
     
     methods(Test)
+        function CheckForHardware(testCase, AD936xClassTx, ...
```

Comment:
Why not do this at within the test setup methods?

Reply:
Addressed by the latest commit. ClassSetupParameter properties block helps overcome issues I was facing earlier with using TestClassSetup  methods block.